docusaurus.config.js

```diff
@@ -396,6 +414,8 @@ module.exports = {
               banner: "none",
             },
           },
+          docLayoutComponent: "@theme/DocPage",
+          docItemComponent: "@theme/ApiItem",
```

Comment:
These overrides are applied to the entire main docs instance, so that API Explorer pages can render properly. I have not found any regressions caused by overriding these for other pages, but we should keep an eye out for that happening.